### DISCUSSION TITLE
Request for enhancement

### DISCUSSION THREAD

**Discussion by rickgartner** [alandtse/alexa_media_player#1400] [Ideas]
I would like to see “Audacy“ added as a media source similar to Spotify or tunein.  This is a media source that has recently added to Amazon in the USA.  I could help with testing if needed.

**Comment by alandtse** [comment]
We don't populate or control the media sources at all. You just need to figure out the call for the service and can update the [wiki](https://github.com/custom-components/alexa_media_player/wiki#examples) if it works.

**Reply by rickgartner** [reply]
Ok, good to know. I thought it was part of the code, but it sounds like I can use any media source.  Thanks
